studio/deskStructure.ts

```diff
@@ -160,10 +161,22 @@ const pagesSection = (S: StructureBuilder) =>
   S.listItem()
     .title("Pages")
     .icon(ProjectsIcon)
-    .child(S.documentTypeList(pageBuilderID).title("Pages"));
+    .child(
+      S.documentTypeList(pageBuilderID)
+        .title("Pages")
+        .child((documentId) =>
+          S.document()
+            .documentId(documentId)
+            .schemaType(pageBuilderID)
+            .views([
+              S.view.form(),
+              S.view.component(PreviewIFrame).title("Preview"),
+            ]),
+        ),
+    );
 
 //Section for set pages
-const specialPagesSection = (S: StructureBuilder) =>
+export const specialPagesSection = (S: StructureBuilder) =>
```

Comment:
```suggestion
const specialPagesSection = (S: StructureBuilder) =>
```

Trenger ikkje å eksportere denne, veit ikkje koffor eg la det til i utgangspunktet.